ts/lib/carina.ts

```diff
@@ -63,6 +70,10 @@ export class Carina {
         .waitFor(`subscription:${slug}`, () => {
             return this.socket.execute('livesubscribe', { events: [slug] });
         })
+        .then(res => {
```

Comment:
Move this up inside the waitFor to prevent slug duplication on the subscriptions array